simulator/src/main/java/com/hedera/block/simulator/Constants.java

```diff
@@ -18,8 +18,12 @@
 
 public class Constants {
 
```

Comment:
When I execute a build I see these warnings:
```
> Task :simulator:javadoc
/Users/mattpeterson/repos/block-node-secondary/hedera-block-node/simulator/src/main/java/com/hedera/block/simulator/Constants.java:19: warning: no comment
public class Constants {
       ^
/Users/mattpeterson/repos/block-node-secondary/hedera-block-node/simulator/src/main/java/com/hedera/block/simulator/Constants.java:19: warning: use of default constructor, which does not provide a comment
public class Constants {
       ^
2 warnings
```

Reply:
The `Constants` class should be final and have a private default constructor

Reply:
@mattp-swirldslabs I will be abstracting all constants and utility classes in the new module based on task #233 which is in progress, so I think it is fine that it will remain like this since it will be changed very soon.

Reply:
Ok, but we should be checking to make sure there aren't any "warnings" when we build